javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-playlist-app/styles.css

```
@@ -1,128 +1,253 @@
+:root {
+  --white-color: #fbf9f9;
+  --main-bg-color: #4d4d62;
+  --player-bg-color: #1b1b32;
+  --player-fg-color: #3b3b4f;
+  --player-highlight-color: #f1be32;
+}
+
 *,
-*::before,
-*::after {
-  margin: 0;
-  padding: 0;
+*::after,
+*::before {
   box-sizing: border-box;
 }
 
-:root {
-  --main-background-color: #0a0a23;
-  --gray100: #ffffff;
-  --gray200: #e9e8e8;
-  --golden-yellow: #feac32;
+body {
```

```suggestion
ul {
  margin: 0;
}
body {
```

If we use a `ul` for the playlist then we need to get rid of the stupid default margins that browsers add.